pkg/apis/networkoperator/v1/networkconfig_types.go

```diff
@@ -179,6 +179,9 @@ const (
 
 	// NetworkTypeRaw
 	NetworkTypeRaw NetworkType = "Raw"
+
+	// NetworkTypeOpenShiftSRIOV
+	NetworkTypeOpenShiftSRIOV NetworkType = "OpenShiftSRIOV"
```

Comment:
why is it specifically OpenShift? What's the difference from a regular SR-IOV network that is managed by upstream CNI/DP? Is it the hardcoded openshift.io prefix?

Reply:
There are two slight differences between openshift SRIOV and upstream ones:
1) The resource name advertised via openshift SRIOV DP is called 'openshift.io/sriov'.
2) Limited set of NICs supported in openshift SRIOV DP.
Besides above, there is no major difference on the design and architecture. Openshift 4.0 and 4.1 releases are based on upstream release-v1 branch (both DP and CNI).